docs/source/examples.rst

```diff
@@ -0,0 +1,94 @@
+
+* We discuss the "goal" of the example. By this we mean the standard reward that is built into the environment. It is simple to change the reward function if other goals are desired.
+
+* Occasionally we describe environments as *fully observed*. By this we mean that all relevant pieces of information (speed, relative positions, traffic light states) of the system are available to the controller. *Partially observed* refers to only a subset being available. This is the default configuration of the environment and the set of observations can be customized.
+
+* Each environment has customizable environment/networks parameters that can be used to configure it beyond what is described here. Particularly pertinent parameters are described. Additional parameters can be found by examining the documentation in the relevant scenario and environment files.
+
+* In the figures below, the following key is used for vehicle colors, where AV stands for autonomous vehicle.
+
+.. image:: ../img/key.png
+   :width: 200
+   :align: center
+
+Figure Eight
+------------
+The figure-eight is a closed-loop version of an intersection. The goal is to maximize
+the system-wide velocity for fourteen vehicles,
+which necessitates spacing the vehicles so that they don't
+run into conflicts at the merging points. The scenario is fully observed: all vehicles
+speeds and positions are visible to the controller.
+This scenario is also a benchmark, and has been
+extensively tested at three penetration rates: 1 AV 13 humans, 7 AVs 7 humans, 14 AVs.
+
+The scenario, pictured below,
+is relatively light-weight and can be trained the quickest. It can serve both as a test
+that the training process is working correctly and as a study of the difficulty of controlling
+many vehicles simultaneously.
+
+.. image:: ../img/figure8-corl2018.png
+   :width: 400
+   :align: center
+
+Stabilizing the Ring
+--------------------
+This example builds on the famous spontaneously forming shockwave from Sugiyama (2008).
+In this scenario, human drivers placed on a ring develop a travelling wave that causes
+decelerations and lowers the average velocity of the system. The goal is to train a single
+autonomous vehicle to eliminate the shockwave.
+
+The pre-built run script for running this in human-only mode is `examples/sumo/sugiyama.py`.
+The run scripts in rllib and rllab are `examples/rllib/stabilizing_the_ring.py` and
+`examples/rllab/stabilizing_the_ring.py`. These control environments are partially observed:
+the autonomous vehicle only observes its own distance to the leading vehicle, its speed,
+and the speed of the leading vehicle.
+
+To make this task more difficult, the environment has a configurable parameter, `ring_length`, which
+can be set to a list containing the minimum and maximum ring-size. The autonomous vehicle must
+learn to distinguish these scenarios from each other and pick the appropriate driving behavior.
+
+.. image:: ../img/stabilizing_the_ring.png
+   :width: 400
```

Comment:
Looking at the file the pictures look huge! Part of it might be that they don't all share the same dimensions. Maybe try using `:height:` instead (if that is a thing?). Also unclear if they will look the same on readthedocs, so unsure

Reply:
ah yeah, I actually looked at it in readthedocs, it's not too big.